docs/references/express/overview.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Clerk Express.js SDK
+description: Learn how to integrate Express.js into your Clerk application.
+---
+
+# Clerk Express.js SDK
+
+## Set up Clerk Express.js
+
+<Steps>
+
+### Create a Clerk application
+
+You need to create a Clerk application in your Clerk Dashboard before you can set up Clerk Express.js. For more information, check out our [Set up your application](/docs/quickstarts/setup-clerk) guide.
+
+### Install `@clerk/express`
+
+To get started using Clerk with Express.js, add the [SDK](https://www.npmjs.com/package/@clerk/express) to your project:
+
+<CodeBlockTabs type="installer" options={["npm", "yarn", "pnpm"]}>
+```sh filename="terminal"
+npm install @clerk/express
+```
+
+```sh filename="terminal"
+yarn add @clerk/express
+```
+
+```sh filename="terminal"
+pnpm add @clerk/express
+```
+</CodeBlockTabs>
+
+### Set environment variables
+
+<SignedIn>
+
+Add the following keys to your `.env.local` file. These keys can always be retrieved from the [API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page of your Clerk Dashboard.
+
+</SignedIn>
+
+<SignedOut>
+
+1. Navigate to the Clerk Dashboard.
+2. In the navigation sidebar, select [API Keys](https://dashboard.clerk.com/last-active?path=api-keys).
+3. In the **Quick Copy** section, copy your Clerk publishable and secret key.
+4. Paste your keys into your `.env.local` file.
+
+The final result should resemble the following:
+
+</SignedOut>
+
+  ```sh filename=".env.local"
+  NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY={{pub_key}}
+  CLERK_SECRET_KEY={{secret}}
+  ```
+
+</Steps>
+
+## Available methods
+
+The Clerk Express SDK exports the following:
+
+- `@clerk/backend` - You can use [Clerk's JavaScript Backend SDK](/docs/references/backend/overview) to interact with the Clerk API.
+- `clerkMiddleware()` - The [`clerkMiddleware()`](/docs/references/nextjs/clerk-middleware) helper integrates Clerk authentication into your Next.js application through Middleware.
+- `getAuth()` - The [`getAuth()`](/docs/references/nextjs/get-auth) helper retrieves the authentication state, allowing you to protect your API routes or gather relevant data.
+- `requireAuth()` -
+
+### Express error handlers
+
+Express comes with a default error handler for errors encountered in the middleware chain.
+
+Developers can also implement their own custom error handlers as detailed in the [Express error handling guide](https://expressjs.com/en/guide/error-handling.html). An example error handler can be found above.
```

Comment:
```suggestion
Express comes with a default error handler for errors encountered in the middleware chain. Developers can also implement their own custom error handlers as detailed in the [Express error handling guide](https://expressjs.com/en/guide/error-handling.html). An example error handler can be found above.
```

Reply:
Also I don't see an error handling example above, did we forget to add this?

Reply:
This was removed, no?